### PR TITLE
Update terms and privacy

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Relates to [#525](https://github.com/near/near-discovery/issues/525), [#526](https://github.com/near/near-discovery/issues/526)

Closes [#727](https://github.com/near/near-discovery/issues/727)

This PR introduces new terms and privacy docs which stores in new infura account. All the credentials could be found in 1Password.

Also this PR includes:
- style changes for ToS modal to be aligned with other gateway components;
- new terms and privacy pages;

### CODE REVIEW

File: src/GatewayWrapper.jsx [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+  logOut,
+  tosDomainName,
+  termsCid,
+  privacyCid,
+  recordToC
+} = props;
+
+tosDomainName = tosDomainName ?? "https://near.infura-ipfs.io";
+termsCid = termsCid ?? "QmaE4P6fn2VSTgAJ4gdEmAh7Ywiw9J3TT8aRjMaFLdoDPX";
+privacyCid = privacyCid ?? "QmRiu67GedetzBMwKMfJkrKnFNDSs8KSUDPKEgXKN2hVqU";
```
Comment (shelegdmitriy) [inline src/GatewayWrapper.jsx:13]: Is it worth to store all this data here? I think it's better to store in .env file in near-discovery and pass it to this component like I did [here](https://github.com/near/near-discovery/issues/727) ?

Comment (calebjacob) [reply src/GatewayWrapper.jsx]: I agree, it probably makes most sense to have the gateway pass these values in. I left a comment on your other PR.

File: src/GatewayWrapper.jsx [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+  targetComponent,
+  targetProps,
+  logOut,
+  termsDomainName,
+  privacyDomainName,
+  recordToC
+} = props;
+
+termsDomainName = termsDomainName ?? "https://near.infura-ipfs.io/ipfs/QmaE4P6fn2VSTgAJ4gdEmAh7Ywiw9J3TT8aRjMaFLdoDPX";
+privacyDomainName = privacyDomainName ?? "https://near.infura-ipfs.io/ipfs/QmRiu67GedetzBMwKMfJkrKnFNDSs8KSUDPKEgXKN2hVqU";
```
Comment (shelegdmitriy) [inline src/GatewayWrapper.jsx:11]: If we are good with storing this in .env file then I'll remove this from here

**Comment by shelegdmitriy** [comment]
@charleslavon @calebjacob I think we're good to merge this PR, any objections?

**Review by charleslavon — APPROVED** [review]
Nice setup 👍🏾

**Pull request merged**